src/remark-jsDoc-link.js

```diff
@@ -0,0 +1,36 @@
+const findAndReplace = require('mdast-util-find-and-replace');
+
+/**
+ * A remark plugin that installs
+ * for JSDoc inline `{@link}` and `{@tutorial}` tags.
+ *
+ * This does not handle the `[text]({@link url})` and `[text]({@tutorial url})` forms of these tags.
+ * That's a JSDoc misfeature; just use regular markdown syntax instead: `[text](url)`.
```

Comment:
Hello, @anthony-redFox . 

I have a question about this change.
I wonder why the syntax is a mifeature.
That syntax still exists in the https://jsdoc.app/tags-inline-link.html documentation.